cd/utils/mxnet_base_image.sh

```diff
@@ -22,22 +22,22 @@ mxnet_variant=${1:?"Please specify the mxnet variant as the first parameter"}
 
 case ${mxnet_variant} in
     cu101*)
-    echo "nvidia/cuda:10.1-cudnn7-runtime-ubuntu16.04"
+    echo "nvidia/cuda:10.1-cudnn7-runtime-ubuntu18.04"
```

Comment:
Were you able to test these changes that it works with ubuntu18.04 ?

Reply:
I think the base image is specified in the docker files? here we might be just echoing a string.

Reply:
are those images only for the docker image pipelines?

Reply:
@access2rohit yes I tested it.

@Zha0q1 yes these base images are only for building docker images that get published.